lib/utils.js

```diff
@@ -95,3 +95,62 @@ exports.findAvailablePort = function(app){
   });
 
 }
+
+exports.autoStoreData = function (req, res, next) {
+
+  // Middleware - store any data sent in session
+
+  if (!req.session.data){
+    req.session.data = {};
+  }
+
+  for (var i in req.body){
```

Comment:
Can you add some kind of comment indicating what this is doing? Ie document that if you don't want data saved, add an underscore to the name.